es-spatial-plugin/src/cmr/elasticsearch/plugins/spatial/factory/lfactory.clj

```diff
@@ -2,7 +2,8 @@
   (:import
    (cmr.elasticsearch.plugins SpatialScript)
    (java.util Map)
-   (org.elasticsearch.script DocReader) 
+   (org.apache.logging.log4j Logger LogManager)
```

Comment:
This is not used, can be removed.

Reply:
If this is the case, I would like to see a commented out log line to show how logging is supposed to be done in the plugin.

Reply:
Added